### PR TITLE
Updated contribution guide

### PR DESCRIPTION
**Pull request by thebe14**
Minor typos and rephrasing
Added prompts before shell command examples
Added tip to describe Copy button behavior for shell examples

# Summary

---

See also issues #271 and #272

### CODE REVIEW

**Comment by gwarf** [comment]
Nice, thanks.
But I think we should first finalise discussions on #271 before merging an update of the documentation about this.

**Comment by thebe14** [comment]
Absolutely. I just prepared a PR to update the contributing guide with whatever is decided in #271. For now, I added:

- keep commands and output in the same block
- add prompts for all commands (this implies disabling the linter rule commands-no-output)
- break long commands to multiple lines

I can amend these as necessary, we will merge this when #271 draws a conclusion.

File: content/en/about/_index.md [hunk 1]
```diff
@@ -16,5 +16,5 @@ using [Hugo](https://gohugo.io).
 [EGI Services](https://www.egi.eu/services/) is maintained by the EGI Community
 and coordinated by the
 [EGI Foundation](https://www.egi.eu/about/egi-foundation/). Everyone is invited
-to participate by following the [Contributing Guide](contributing).
+to participate, by following the [Contributing Guide](contributing).
```
Comment (gwarf) [inline content/en/about/_index.md:19]: The comma is redundant, no need for it (confirmed by native English speaker in Slack on #english-language).

Comment (thebe14) [reply content/en/about/_index.md:19]: Fixed

**Pull request closed without merging**